### PR TITLE
Feature/array functions done

### PR DESCRIPTION
**Pull request by FranDRoss**

### CODE REVIEW

File: staff/francesc-destruels/array-functions/expect-1.0.2.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ console.log('%c expect', 'font-weight: bold; font-size: 50px; color: red; text-s
  * 
  * @param {*} actual The actual value to check agains the expected value.
  * @param {*} expected The expected value.
- * @param {boolean} [deep] Indicates wether the interal check comparison must be deep or not (in case of arrays or objects).
```
Comment (lexbarba) [inline staff/francesc-destruels/array-functions/expect-1.0.2.js]: @FranDRoss you are updating a file from Manu. Please update PR where only your files are updated, added or removed. You have also renamed two of his files.

Comment (FranDRoss) [reply staff/francesc-destruels/array-functions/expect-1.0.2.js]: I will do it ASAP

**Comment by lexbarba** [comment]
@FranDRoss there are a few files from this folder stuff/css/ that should not be in here. Please update PR or close this one and create a new and clean one. Thanks.

**Comment by lexbarba** [comment]
@FranDRoss there are a few files from this folder stuff/css/ that should not be in here. Please close this PR and create a new and clean one. Thanks.

**Pull request closed without merging**